src/Compilers/VisualBasic/Portable/Symbols/NamedTypeSymbol.vb

```diff
@@ -1089,6 +1099,102 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         ''' </summary>
         Friend MustOverride Function GetSynthesizedWithEventsOverrides() As IEnumerable(Of PropertySymbol)
 
+        Friend ReadOnly Property AllRequiredMembers As ImmutableSegmentedDictionary(Of String, Symbol)
+            Get
+                TryCalculateRequiredMembersWithCache(Nothing)
+                Debug.Assert(Not _lazyRequiredMembers.IsDefault)
+                Return _lazyRequiredMembers
```

Comment:
>_lazyRequiredMembers [](http://example.com/codeflow?start=23&length=20)

It looks like here we can return value of ```s_requiredMembersErrorSentinel```. Is this intentional? This feels like an exposure of an implementation detail, which might really confuse some consumers because content of ```s_requiredMembersErrorSentinel``` is "malformed". Would it be better to return an empty dictionary instead of ```s_requiredMembersErrorSentinel```?